src/hooks/http.test.js

```diff
@@ -838,6 +875,49 @@ describe('http hook', () => {
     expect(span.info.httpInfo.request.headers[TRACESTATE_HEADER_NAME]).toBeDefined();
   });
 
+  test('wrapHttpLib - check request - headers exists', () => {
```

Comment:
What is the purpose of the test? Verifying that `TRACESTATE_HEADER_NAME` exists? Please add it in the description of the test. Same regarding the next test.